girder/__init__.py

```diff
@@ -104,7 +104,7 @@ def __init__(self, stream, logger, level):
                     and (callable(getattr(stream, key, None))
                          or isinstance(
                              getattr(stream, key, None),
-                             (six.binary_type, six.string_types, six.integer_types, bool)))):
```

Comment:
I would have assumed this would not be valid, but it turns out `isinstance` flattens sub-tuples, which is kind of gross.